tests/test_checks.py

```diff
@@ -0,0 +1,142 @@
+from django.test import SimpleTestCase
+from django.test.utils import override_settings
+from django.core import checks
+
+from corsheaders.conf import corsheaders_settings
+from corsheaders.checks import check_settings
+
+from corsheaders.defaults import default_headers, default_methods
+
+class SettingsTests(SimpleTestCase):
+
```

Comment:
one thing these tests don't do is test that `manage.py check` really will call `check_setttings`. You could add a test that does `call_command('check')` with one piece of configuration broken and ensure that it raises `SystemCheckError`